src/components/FoodForm.js

```diff
@@ -109,7 +189,9 @@ const FoodForm = () => {
 					);
 				})}
 			</StlyedList>
-			<div>total calories consumed today:{calories}</div>
+			<StyledCalories>
+				<div>total calories consumed today: {calories}</div>
```

Comment:
A `p` element is more semantic than a `div` element.

Reply:
you are right, thanks.